### ISSUE TITLE
Add different electric cars to simulation 

### ISSUE THREAD

**Issue by NofTheGreek** [matsim-org/matsim-code-examples#425]
Hi, i'm trying to add some well known electric vehicles to my Matsim project ( Nissan Leaf, Tesla Model 3 etc ). Does anyone know how to do that? I need to add those cars to the evehicles.xml file but after that how can I assign them to the population and get the consumption rates? Thank you in advance.

**Comment by michalmac** [comment]
Create a list of e-vehicles, each having the same ID as the corresponding matsim vehicle. Typically, the latter are auto-generated and use person IDs as their IDs.

**Comment by tschlenther** [comment]
@michalmac 
Actually, i planned to address you eventually with kind of the same topic. 
It would be very nice if the ev contrib would rely on standard matsim vehicle types. 
Similar to the discussion #988

I wrote an adapter ElectricVehicleSpecificationProvider for one of my project as an intermediate solution.
If you want, you can have a look at `MATSimVehicleWrappingEVSpecificationProvider` on `ev` branch of matsim-berlin (see https://github.com/matsim-scenarios/matsim-berlin/tree/ev). It needs another class `EVUtils` though.

If you want, have a look and tell me what you think. We can discuss the bigger picture later. Initially planned to come up with this as soon as the code project is finished, but as now this issue came up i thought i take the chance to jump in.

**Comment by michalmac** [comment]
Yes, similar issue as in DVRP. I would like to read in vehicles from the standard matsim vehicle file and translating them into electric vehicles (maybe wrapping, maybe copying, not sure yet). I will have a look at it next week.

**Comment by NofTheGreek** [comment]
> Create a list of e-vehicles, each having the same ID as the corresponding matsim vehicle. Typically, the latter are auto-generated and use person IDs as their IDs.

So i create a evehiclesxml file with as many vehicles as population and the ID from evehicles files wil be automatically be assigned to population IDs ?
